example/src/components/TransactionTable/TransactionTable.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {Paper, Table, TableContainer, TableCell,
+    TableRow, TableHead, TableBody} from '@material-ui/core/';
+
+export const TransactionTable = () => {
+
+    function createData(id: string, from: string, to: string, amount: number, token: string) {
+        return { id, from, to, amount, token };
+      }
+    const rows = [
+        createData('21341', "0XDC25EF3F5B8A...", "0XDC25EF3F5B8A...", 0.34, "ETH"),
+        createData('67893', "0XDC25EF3F5B8A...", "0XDC25EF3F5B8A...", 0.11, "ETH"),
+        createData('34673', "0XDC25EF3F5B8A...", "0XDC25EF3F5B8A...", 0.09, "ETH"),
+        createData('08763', "0XDC25EF3F5B8A...", "0XDC25EF3F5B8A...", 0.46, "ETH"),
+        createData('12309', "0XDC25EF3F5B8A...", "0XDC25EF3F5B8A...", 1.39, "ETH"),
+      ];
+
+    return (
+        <TableContainer className="transtaction-table" component={Paper}>
+            <Table 
```

Comment:
checkout docs and use options provided to table related classes to make it pretier